src/System.Net.Sockets/src/System/Net/Sockets/UDPClient.cs

```diff
@@ -59,38 +38,27 @@ public UdpClient(AddressFamily family)
             CreateClientSocket();
         }
 
-        // bind specific port, arbitrary IP
-        /// <devdoc>
-        ///    <para>
-        ///       Creates a new instance of the UdpClient class that communicates on the
-        ///       specified port number.
-        ///    </para>
-        /// </devdoc>
-        /// We should obsolete this. This also breaks IPv6-only scenarios.
-        /// But fixing it has many complications that we have decided not
-        /// to fix it and instead obsolete it post-Orcas.
+        // Creates a new instance of the UdpClient class that communicates on the
+        // specified port number.
+        //
+        // NOTE: We should obsolete this. This also breaks IPv6-only scenarios.
+        // But fixing it has many complications that we have decided not
+        // to fix it and instead obsolete it later.
         public UdpClient(int port) : this(port, AddressFamily.InterNetwork)
```

Comment:
@SidharthNabar @josguil: Is this considered for removal? The comments are pointing out that this ctor should be obsolete.